src/statement/create-table.js

```diff
@@ -150,45 +149,46 @@ class CreateTable extends Statement {
    * @return String The SQL columns definition.
    */
   _buildDefinition() {
-    var result = [];
-    var columns = this._parts.columns;
-    var constraints = this._parts.constraints;
-    var primary;
+    let primary;
+    const result = [];
+    const columns = this._parts.columns;
+    const constraints = this._parts.constraints;
+    const dialect = this.dialect()
```

Comment:
is this called multiple times in these `_build*` functions? I'd be curious to know the performance impact of that function

Reply:
it doesn't look like might it's a getter with a shortcut